### PR TITLE
Plugin E2E: Replace magic strings with selectors

### PR DESCRIPTION
**Pull request by sunker**
**What this PR does / why we need it**:

In this PR I'm replacing all remaining magic strings with proper selectors defined in `e2e-selectors`. 

**Which issue(s) this PR fixes**:

Fixes #
Part of https://github.com/grafana/plugin-tools/issues/576

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/plugin-e2e/tests/datasource/variables/customVariableEditor.spec.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ test('should load resources and display them as options when clicking on an inpu
   await variableEditPage.datasource.set(provision.datasources?.[0]!.name!);
   await page.getByLabel('Schema').click();
   await expect(variableEditPage.getByTestIdOrAriaLabel('Select option')).toContainText(REDSHIFT_SCHEMAS);
-  await page.keyboard.press('Escape');
+  await page.keyboard.press('Enter');
```
Comment (academo) [inline packages/plugin-e2e/tests/datasource/variables/customVariableEditor.spec.ts:23]: Is this change intended?

**Review by academo — APPROVED** [review]

File: packages/plugin-e2e/tests/datasource/variables/customVariableEditor.spec.ts (hunk 1 above)
Comment (sunker) [reply packages/plugin-e2e/tests/datasource/variables/customVariableEditor.spec.ts:23]: Yes - In Grafana 9.2.x (and earlier), hitting Escape takes you away from dashboard settings and back to the dashboard start page. So needed some other command that can move focus away from the select, but still stay on the same page.
